src/modules/launcher/Wox.Plugin/Common/DefaultBrowserInfo.cs

```diff
@@ -157,32 +157,42 @@ public static void Update()
                         SearchEngineUrl = null;
                     }
 
-                    string programLocation =
-
-                        // Resolve App Icon (UWP)
-                        GetRegistryValue(
-                            $@"HKEY_CLASSES_ROOT\{progId}\Application",
-                            "ApplicationIcon")
+                    // Get the path to the browser's executable:
+                    // for example "C:\Program Files\SomeCompany\SomeBrowser\Application\launcher.exe" --single-argument %1
+                    string programLocation = GetRegistryValue($@"HKEY_CLASSES_ROOT\{progId}\shell\open\command", null);
+                    if (string.IsNullOrEmpty(programLocation))
+                    {
+                        throw new ArgumentOutOfRangeException(
```

Comment:
will this crash PowerToys

Reply:
No, this is happening in `try...catch` block.